### ISSUE TITLE
Add directions

### ISSUE THREAD

**Issue by planemad** [publicmap/publicmap.github.io#4]
Ref: https://www.mapbox.com/mapbox-gl-js/example/mapbox-gl-directions/

**Comment by SrihariThalla** [comment]
This is good. Should not take much time.

Personally, I'm very much interested in checking out how it differs from Google!

On a side note, I don't think Mapbox has a public map like Google Maps [and an app]. Is that correct? (Probably the business models are different!) This repo would be a good choice to start with!

**Comment by planemad** [comment]
> don't think Mapbox has a public map like Google Maps

Thats right, there's no consumer facing app ready to use and developers needs to build it using the various APIs and libraries available. Which is why publicmap 😸
